packages/snaps-controllers/src/interface/utils.test.tsx

```diff
@@ -1222,3 +1223,13 @@ describe('getDefaultAsset', () => {
     );
   });
 });
+
+describe('isStatefulComponent', () => {
```

Comment:
Can you add a test with a stateful component nested inside a non-stateful component?

Reply:
`isStatefulComponent` does not walk the JSX tree

Reply:
I get that, but we should make sure that's the expected behaviour by having a test for it.

Reply:
Oh okay ! Will add then